graylog2-server/src/main/java/org/graylog2/rest/resources/users/UsersResource.java

```diff
@@ -152,11 +179,7 @@ public UserList listUsers() {
 
         // among all active sessions, find the last recently used for each user
         //noinspection OptionalGetWithoutIsPresent
```

Comment:
This `noinspection` comment should be moved to the `getLastSessionForUser()` method.